include/pdal/PDALUtils.hpp

```diff
@@ -106,7 +106,7 @@ inline Everything extractDim(Extractor& ext, Dimension::Type type)
 {
     using Type = Dimension::Type;
 
-    Everything e;
+	Everything e{ 0 };
```

Comment:
This is a bit of performance sensitive code.  The optimizer may eliminate the unnecessary initialization, but I'm not sure.  Certainly this is an unnecessary initialization, though.